docs/migrating.storybook.mdx

```diff
@@ -1,91 +1,293 @@
 import { Meta } from '@storybook/blocks';
 
-import { PlacementMappingsTable } from './components/PlacementMappingsTable';
+import { PlacementMappingsTable } from './components/migration/PlacementMappingsTable';
+import { InvertedPropMigrationTable } from './components/migration/InvertedPropMigrationTable';
+import { IconMappingsTable } from './components/migration/IconMappingsTable';
+import { CssVariablesMigrationTable } from './components/migration/CssVariablesMigrationTable';
 
-<Meta title="Migrating" />
+<Meta title="Migration to v2" />
 
-# Migrating
+# From v1 to v2
 
-Migrate to the next Wave major version with ease.
+Prerequisites:
 
-## Codemods
+-   your project must use React@18;
+-   React is mounted to the DMT using [new `createRoot` API](https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis).
 
-Codemods are scripted transformations that you can run on your code to take care of most of the repetitive work involved
-in upgrading from one major version to the next. The codemod instructions for each major release assume that your code
-has already been upgraded to the previous major release. If this is not the case, you should run the codemods for all
-previous major versions first. Beware that codemods are not 100% fool-proof and may break your code.
+## 1️⃣ Connect Classic colors
 
-If you're working in a typescript environment, you need to add `--parser=tsx` to the command in each codemod command to
-allow jscodeshift to parse your typescript/jsx files.
+Version 2 provides two [color themes](?path=/docs/essentials-colors--docs) of the interface.
```

Comment:
nit: I find it a bit confusing to read "themes of the interface", is it referring to UI? Can we remove "of the interface" or replace it with UI?